hudi-common/src/main/java/org/apache/hudi/BaseHoodieTableFileIndex.java

```diff
@@ -314,9 +314,7 @@ protected HoodieTimeline getActiveTimeline() {
   private Object[] parsePartitionColumnValues(String[] partitionColumns, String partitionPath) {
     Object[] partitionColumnValues = doParsePartitionColumnValues(partitionColumns, partitionPath);
     if (shouldListLazily && partitionColumnValues.length != partitionColumns.length) {
-      throw new HoodieException("Failed to parse partition column values from the partition-path:"
-          + " likely non-encoded slashes being used in partition column's values. You can try to"
-          + " work this around by switching listing mode to eager");
+      LOG.warn(">>> PartitionColumns: " + partitionColumns + "  PartitionValues: " + partitionColumnValues);
```

Comment:
So I assume we still need to fail here instead of printing the warning and letting it return?